OtripleS.Web.Api/Controllers/StudentExamFeesController.cs

```diff
@@ -0,0 +1,199 @@
+
+                return Conflict(innerMessage);
+            }
+            catch (StudentExamFeeValidationException studentExamFeeValidationException)
+            {
+                string innerMessage = GetInnerMessage(studentExamFeeValidationException);
+
+                return BadRequest(innerMessage);
+            }
+            catch (StudentExamFeeDependencyException studentExamFeeDependencyException)
+            {
+                return Problem(studentExamFeeDependencyException.Message);
+            }
+            catch (StudentExamFeeServiceException studentExamFeeServiceException)
+            {
+                return Problem(studentExamFeeServiceException.Message);
+            }
+        }
+
+        [HttpGet]
+        public ActionResult<IQueryable<StudentExamFee>> GetAllStudentExamFees()
+        {
+            try
+            {
+                IQueryable storageStudentExamFee =
+                    this.studentExamFeeService.RetrieveAllStudentExamFees();
+
+                return Ok(storageStudentExamFee);
+            }
+            catch (StudentExamFeeDependencyException studentExamFeeDependencyException)
+            {
+                return Problem(studentExamFeeDependencyException.Message);
+            }
+            catch (StudentExamFeeServiceException studentExamFeeServiceException)
+            {
+                return Problem(studentExamFeeServiceException.Message);
+            }
+        }
+
+        //[HttpGet("{studentExamFeeId}")]
+        //public async ValueTask<ActionResult<StudentExamFee>> GetStudentExamFeeAsync(Guid studentExamFeeId)
+        //{
+        //    try
+        //    {
+        //        StudentExamFee storageStudentExamFee =
+        //            await this.studentExamFeeService.RetrieveStudentExamFeeByIdAsync(studentExamFeeId);
+
+        //        return Ok(storageStudentExamFee);
+        //    }
+        //    catch (StudentExamFeeValidationException studentExamFeeValidationException)
+        //        when (studentExamFeeValidationException.InnerException is NotFoundStudentExamFeeException)
+        //    {
+        //        string innerMessage = GetInnerMessage(studentExamFeeValidationException);
+
+        //        return NotFound(innerMessage);
+        //    }
+        //    catch (StudentExamFeeValidationException studentExamFeeValidationException)
+        //    {
+        //        string innerMessage = GetInnerMessage(studentExamFeeValidationException);
+
+        //        return BadRequest(innerMessage);
+        //    }
+        //    catch (StudentExamFeeDependencyException studentExamFeeDependencyException)
+        //    {
+        //        return Problem(studentExamFeeDependencyException.Message);
+        //    }
+        //    catch (StudentExamFeeServiceException studentExamFeeServiceException)
+        //    {
+        //        return Problem(studentExamFeeServiceException.Message);
+        //    }
+        //}
```

Comment:
@viralpandya you can't have that in here.

Reply:
Resolved